common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -335,7 +336,14 @@
             DiscussionThreadListView.prototype.renderThread = function(thread) {
                 var threadCommentCount = thread.get('comments_count'),
                     threadUnreadCommentCount = thread.get('unread_comments_count'),
-                    neverRead = !thread.get('read') && threadUnreadCommentCount === threadCommentCount,
+                    // @TODO: On the profile page, thread read state for the viewing user is not accessible via the API.
+                    // In this case, neverRead is set to false regardless of read state returned by the API.
+                    // Fix this when the Discussions API can support this query.
+                    neverRead = (
```

Comment:
Nit: I'd prefer that a variable called ``neverRead`` actually have the value as to whether or not the thread has been read. If we are not showing that information, that seems different to me. I would probably pass ``showReadState`` to the template too, and let it decide how to handle it.